xc7/utils/prjxray_edge_library.py

```diff
@@ -1647,6 +1675,24 @@ def annotate_pin_feeds(conn, ccio_sites):
     cur.execute("SELECT pkey FROM segment WHERE name = ?", ("OUTPINFEED", ))
     outpinfeed_pkey = cur.fetchone()[0]
 
+    cur.execute(
```

Comment:
Should probably make a simply name -> pkey map, and convert these into a loop to build the map.